### ISSUE TITLE
New definition, rename & parent change: GO:0015341 zinc efflux active transmembrane transporter activity

### ISSUE THREAD

**Issue by hattrill** [geneontology/go-ontology#27494]
Update to show that this is a Zn/H antiporter. PMID:35226479 and https://tcdb.org/search/result.php?tc=2.A.4.7.1 describes the mechanism as antiporter Zn2+/H+ (using the proton motive force for H+ movement). 

* **GO term ID and label for which you request a definition update**
* GO:0015341 zinc efflux active transmembrane transporter activity

* **New parent**
* is_a GO:0140826 zinc:proton antiporter activity

* **Rename**
* zinc efflux antiporter activity

* **New proposed definition**
* Enables the transfer of a zinc ion or zinc ions from the inside of the cell to the outside of the cell across a membrane  according to the reaction H+(out) + Zn2+(in) = H+(in) + Zn2+(out). The activity is driven by proton motive force. 

* **Reference(s)**
 PMID:35226479

**Comment by hattrill** [comment]
#27495

**Comment by sjm41** [comment]
Hi @hattrill

If this is specifically for efflux "from the inside of the cell to the outside of the cell" then I think:
i) the reaction direction should be specified: H+(out) + Zn2+(in) => H+(in) + Zn2+(out)
ii) "across a membrane" should be specified as "across the cell membrane".

?

**Comment by hattrill** [comment]
These are just modifications to the existing patterns, so would have to defer to an editor.

"Enables the transfer of a zinc ion or zinc ions from the inside of the cell to the outside of the cell across a membrane: Zn2+(out) = Zn2+(in). The activity is driven by proton motive force." 

I can see some => used in some 21 reaction defintions in GO, so is possible.

"across a membrane" is the pattern for  GO:0015562 efflux transmembrane transporter activity.

"according to the reaction" comes from another transporter definition style that has been used for transporters e.g. GO:0140929 - that was my only filigree.

**Comment by sjm41** [comment]
Yep, I think that all such efflux transporters should have the pattern I suggest!

**Comment by lbreuza** [comment]
> Hi @hattrill
> 
> If this is specifically for efflux "from the inside of the cell to the outside of the cell" then I think: i) the reaction direction should be specified: H+(out) + Zn2+(in) => H+(in) + Zn2+(out) ii) "across a membrane" should be specified as "across the cell membrane".
> 
> ?

Shouldn't this term be deleted as it  includes more than the MF with a localization (plasma membrane) and some topology (in/out). 

We could simply use GO:0140826=zinc:proton antiporter activity
Enables the transfer of zinc from one side of a membrane to the other according to the reaction: H+(out) + Zn2+(in) = H+(in) + Zn2+(out).
You could combine this with an annotation is_active_in 'plasma membrane'

and use the BP:
GO:0140882  zinc export across plasma membrane
The directed movement of zinc ions from a cell, into the extracellular region. PMID:17355957
OR
some extensions to state where the zinc comes from and is ending in

**Comment by hattrill** [comment]
Hi @lbreuza -  I just have the aim for the carrier mechanism to be captured correctly.

There is only one EXP annotation to this term, I think that the group could just capture it as zinc:proton antiporter activity. We would  have to just use = not => as there would be no directionality on the reaction.
But it would solve the issue of the mechanism issue for me.

However, there's a whole branch of efflux transporter activity: GO:0015562 efflux transmembrane transporter activity
So this would be of larger scope that I am not going to touch right now.

**Comment by lbreuza** [comment]
Hi Helen,
I understand perfectly you do not want to dive in all that cleaning but I wanted to have your thoughts on the need and use of such detailed MF terms that try to encapsulate different types of additional information with respect to the use of extensions and BP to provide more details on the where, how and why the antiport reaction occurs.

**Comment by hattrill** [comment]
@lbreuza I am with you on that. I feel that efflux would be better as part of a process.
Having to specificy 'efflux' on an activity seems like too much detail for GO.
So, we could start with this term and just merge GO:0015341 into GO:0140826 zinc:proton antiporter activity.
